server/controllers/abc.py

```diff
@@ -5,9 +5,6 @@ class DatabaseDriver:
     def __init__(self, database_client=None):
         self.client = database_client
 
-    def get(self, data):
-        raise NotImplementedError()
-
```

Comment:
ngl it's kind of funny to not have a get function for a database driver :sweat_smile: